docs/.vuepress/styles/palette.styl

```
@@ -1,4 +1,5 @@
 $accentColor = #DD0B78
+$accentDarkColor = ##e50a6c
```

It looks like the site isn't building due to the extra hash here:
```suggestion
$accentDarkColor = #e50a6c
```